patches/tModLoader/Terraria/ModLoader/DamageClassLoader.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Terraria.ModLoader
+{
+	public static class DamageClassLoader
+	{
+		public static int DamageClassCount => damageClasses.Count;
+
+		internal static List<ModDamageClass> damageClasses = new List<ModDamageClass>();
+
+		internal static bool DamageClassExists(ModDamageClass damageClass) => damageClass == null ? false : damageClasses.Contains(damageClass);
```

Comment:
This is just a glorified null check. I don't think there's a need for Contains calls everywhere.